index.js

```diff
@@ -2015,6 +2015,23 @@
     return new RegExp(source, flags);
   });
 
+  //# regexEscape :: String -> String
+  //.
+  //. Takes a string which may contain regular expression metacharacters,
+  //. and returns a string with those metacharacters escaped.
+  //.
+  //. Properties:
+  //.
+  //.   - ∀ `s` ∈ String: `S.test(S.regex('', S.regexEscape(s)), s)`
```

Comment:
> Properties:
>
>   - ∀ `s` ∈ String: `S.test(S.regex('', S.regexEscape(s)), s)`

Is this a sensible way to express this property? How could the formatting be improved? Should we use this approach in the Ramda documentation as well?

/cc @CrossEye, @raine, @scott-christopher